backend/curation/api/v1/curation/collections/common.py

```diff
@@ -304,6 +304,7 @@ class EntityColumns:
         "development_stage",
         "cell_type",
         "cell_count",
+        "primary_cell_count",
```

Comment:
I think we actually need to add a None default value for getattr [here](https://github.com/chanzuckerberg/single-cell-data-portal/pull/5705/files#diff-26c746d9f1e6e25a74bd8918b8773e6352c44219da15aeb38662790fbc721608R216) to support the missing primary cell counts while we haven't backfilled